frontend/components/SwipeListElement/index.js

```diff
@@ -0,0 +1,109 @@
+import React, { useRef } from 'react';
+import { Animated, StyleSheet, Text, TouchableOpacity, View } from 'react-native';
+import { Swipeable, RectButton } from 'react-native-gesture-handler';
+import { Button, withTheme } from 'react-native-paper';
+import ListElement from '../ListElement';
+// const AnimatedView = Animated.createAnimatedComponent(View);
+
+// const renderRightActions = (progress, dragX) => {
+// 	const scale = dragX.interpolate({
+// 		inputRange: [-80, 0],
+// 		outputRange: [1, 0],
+// 		extrapolate: 'clamp',
+// 	});
+// 	return (
+// 		<RectButton style={styles.rightAction}>
+// 			<AnimatedView style={[styles.actionItems, { transform: [{ scale }] }]} />
+// 		</RectButton>
+// 	);
+// };
+
+const DashListElement = withTheme(({ iconName, primaryText, secondaryText, theme }) => {
```

Comment:
Not sure where this would go, but seems like when I swipe left, the rest of the text get's pushed off screen. I'll send a screen shot in our thread. I dunno how much work this'll be but if it's a quick fix, I say let's do it.